jwst/regtest/test_NIRcam_level2a.py

```diff
@@ -0,0 +1,66 @@
+""" Test for the detector1 pipeline using NIRCam data. This should take an uncal
+    file and generate the stage 1 FITS files (rate, ratints, and trapsfilled)."""
+import os
+from glob import glob
+import pytest
+
+from astropy.io.fits.diff import FITSDiff
+from jwst.pipeline import Detector1Pipeline
+
+@pytest.mark.bigdata
+def test_NIRcam_stage1_rate(rtdata, fitsdiff_default_kwargs, _jail):
+    """
+        Check to make sure that the input and outout files exist and that they are not the
+        same file. Then verify that the pipeline output matches the truth file.
+    """
+
+    rtdata.get_data("nircam/test_detector1pipeline/jw82500001003_02101_00001_NRCALONG_uncal.fits")
+    Detector1Pipeline.call(rtdata.input, steps={'jump': {'rejection_threshold': 150}},
+                           save_results=True)
+    rtdata.output = "jw82500001003_02101_00001_NRCALONG_rate.fits"
+
+    rtdata.get_truth("truth/nircam/test_detector1pipeline/jw82500001003_02101_00001_NRCALONG_rate.fits")
+    assert rtdata.output != rtdata.truth
+
+    diff = FITSDiff(rtdata.output, rtdata.truth, **fitsdiff_default_kwargs)
+    assert diff.identical, diff.report
+
+@pytest.fixture(scope="module")
+def run_pipeline(jail, rtdata_module):
+    """Run calwebb_detector1 pipeline on NIRCam uncalibrated data. For these data the
+       rejection_threshold is increased from the default of 4 to 150 to reduce the
+       runtime to a managable level."""
+    rtdata = rtdata_module
+    rtdata.get_data("nircam/test_detector1pipeline/jw82500001003_02101_00001_NRCALONG_uncal.fits")
+
+    Detector1Pipeline.call(rtdata.input, steps={'jump': {'rejection_threshold': 150}},
```

Comment:
As I mentioned in comments to #4356 it's not good to be running the tests via either the .call() or .run() methods, without using a cfg file, because the cfg files often have several params set to skip certain steps for certain modes of data. In the case of calwebb_detector1.cfg, it is hardwired to always skip the ``ipc`` step. By using the .call() method to run the pipeline, the ``ipc`` step will be applied, resulting in non-standard output.

You still have the ability to pass params as arguments when running from the command line. For example, this could be done with:
```
args = ["calwebb_detector1.cfg", rtdata.input, "--steps.jump.rejection_threshold=150", "--save_results=True"]
Step.from_cmdline(args)
```
of course you have to first collect the cfg file(s) or just include the calwebb_detector1.cfg file in the input data directory on artifactory.

Also, regarding the jump.rejection_threshold, we don't want to waste a lot of unnecessary time by having lots of bogus jumps flagged, but at the same time we also want to test the ability to detect and handle at least *some* flagged jumps. So take a look at the log from the jump step to verify that at least some (> 0) pixels got flagged as containing jumps. If not, you may need to decrease the rejection_threshold a bit till you find an appropriate "sweet spot."